cradle-core/src/main/java/com/exactpro/cradle/messages/MessageBatchToStore.java

```diff
@@ -101,8 +101,8 @@ public StoredMessage addMessage(MessageToStore message) throws CradleStorageExce
 		Instant now = Instant.now();
 		if (message.getTimestamp().isAfter(now))
 			throw new CradleStorageException(
-					"Message timestamp '" + TimeUtils.toLocalTimestamp(message.getTimestamp()) +
-							"' It cannot be greater than current '" + TimeUtils.toLocalTimestamp(now) + "'");
+					"Message timestamp is (" + TimeUtils.toLocalTimestamp(message.getTimestamp()) +
+							") It cannot be greater than current timestamp (" + TimeUtils.toLocalTimestamp(now) + ")");
```

Comment:
"Message timestamp (" + TimeUtils.toLocalTimestamp(message.getTimestamp()) +
							") is greater than current timestamp (" + TimeUtils.toLocalTimestamp(now) + ")");